### ISSUE TITLE
Max2Babylon - Copy Textures to Ouput disabled + GLB = errors

### ISSUE THREAD

**Issue by WVNicolayeff** [BabylonJS/Exporters#329]
Hey there,

Textures getting into GLB files seems to depend on the textures existing in the export folder. Not necessarily a huge issue on it's own, but it seems like it's waiting to check whether they exist until after the geometry (or some other data?) is exported.

In this case I was testing with an architecture file I pulled in from Revit, so exports were pretty slow. I'd expected that if it were GLB then I wouldn't need to have the textures also show up in the folder, similar to FBX's embed media setting. However, unchecking the _Copy textures to output_ box caused it to spend a few minutes exporting the file only to have an error when it got to the materials. After some troubleshooting I discovered the need for the texture files being there when the file is written.

Ideally it would be great it if it wrote them to some kind temp folder when exporting that would be cleared automatically afterwards, but for a short term solution it seems like a quick check for missing textures could be done earlier on in the process.

**Comment by deltakosh** [comment]
Good catch! We will make sure to fix that asap

**Comment by kcoley** [comment]
Hi @WVNicolayeff.  I will make an update to the exporter to not write out textures to the output directory when generating a glb file.  Is this what you are looking for?

**Comment by deltakosh** [comment]
Seems to be the same problem

**Comment by kcoley** [comment]
@WVNicolayeff @DidierBeloola , I just made an update to the Max2Babylon exporter.  Exporting to glb will now only export the glb file to the output directory instead of also including the textures used to create the file.  I also changed the "copy textures to output" checkbox to "write textures" and added another checkbox, "overwrite textures", which gives you the option to overwrite the textures in the output directory.

Let me know if this solves your issue!
